src/resources/help/en_US/relnotes3.3.0.html

```diff
@@ -0,0 +1,111 @@
+  <li><a href="#Advanced_Features">Features for advanced users</a></li>
+  <li><a href="#GUI_Changes">Changes in GUI behavior</a></li>
+  <li><a href="#File_format">Changes to file format readers</a></li>
+  <li><a href="#Plot_changes">Changes to plots</a></li>
+  <li><a href="#Operator_changes">Changes to operators</a></li>
+  <li><a href="#Expression_changes">Changes to expressions</a></li>
+  <li><a href="#Query_changes">Changes to picks and queries</a></li>
+  <li><a href="#Bugs_fixed">Other bug fixes</a></li>
+  <li><a href="#Build_features">Changes for build_visit</a></li>
+  <li><a href="#Dev_changes">Changes for VisIt developers</a></li>
+</ul>
+
+<a name="General_features"></a>
+<p><b><font size="4">General features added in version 3.2</font></b></p>
+<ul>
+  <li>General Feature 1</li>
+  <li>General Feature 2</li>
+</ul>
+
+<a name="Advanced_features"></a>
+<p><b><font size="4">Advanced features added in version 3.2</font></b></p>
+<ul>
+  <li>Advanced Feature 1</li>
+  <li>Advanced Feature 2</li>
+</ul>
+
+<a name="GUI_changes"></a>
+<p><b><font size="4">Changes in GUI behavior for version 3.2</font></b></p>
+<ul>
+  <li>GUI Change 1</li>
+  <li>GUI Change 2</li>
+</ul>
+
+<a name="File_format"></a>
+<p><b><font size="4">File format reader changes in version 3.2</font></b></p>
+<ul>
+  <li>Enhanced the Mili reader to support setting the global integration point for element sets. This can be accomplished by changing the default open options for Mili within the Plugin Manager. Available options are "Inner", "Middle", and "Outer".<li>
+</ul>
+
+<a name="Plot_changes"></a>
+<p><b><font size="4">Changes to VisIt's plots in version 3.2</font></b></p>
+<ul>
+  <li>Plot Change 1</li>
+  <li>Plot Change 2</li>
+</ul>
+
+<a name="Operator_changes"></a>
+<p><b><font size="4">Changes to VisIt's operators in version 3.2</font></b></p>
+<ul>
+  <li>Operator Change 1</li>
+  <li>Operator Change 2</li>
+</ul>
+
+<a name="Expression_changes"></a>
+<p><b><font size="4">Changes to VisIt's expression language in version 3.2</font></b></p>
+<ul>
+  <li>Expression Change 1</li>
+  <li>Expression Change 2</li>
+</ul>
+
+<a name="Query_changes"></a>
+<p><b><font size="4">Changes to VisIt's picks and queries in version 3.2</font></b></p>
+<ul>
+  <li>Query Change 1</li>
+  <li>Query Change 2</li>
+</ul>
+
+<a name="Bugs_fixed"></a>
+<p><b><font size="4">Other bugs fixed in version 3.2</font></b></p>
+<ul>
+  <li>Bug Fix 1</li>
+  <li>Bug Fix 2</li>
+</ul>
+
+<a name="Build_features"></a>
+<p><b><font size="4">Build features added in version 3.2</font></b></p>
+<ul>
+  <li>Build Feature 1</li>
+  <li>Build Feature 2</li>
+</ul>
+
+<a name="Dev_changes"></a>
+<p><b><font size="4">Changes for VisIt developers in version 3.2</font></b></p>
+<ul>
+  <li>Developer Change 1</li>
+  <li>Developer Change 2</li>
+</ul>
+
+<p>Click the following link to view the release notes for the previous version
+of VisIt: <a href=relnotesX.Y.Z.html>X.Y.Z</a>.</p>
```

Comment:
X.Y.Z should be changed to 3.1.4

Reply:
actually, X.Y.Z should be 3.2.Z (since we don't know what Z will be for 3.2 until we are ready to release 3.3.0)

Reply:
Yeah, that's why I left this one blank. I can change it to 3.2.Z if that makes more sense (as long as it doesn't go undetected).

Reply:
Fixed.